astropy/io/fits/fitstime.py

```diff
@@ -231,7 +319,7 @@ def _convert_global_time(table, global_info):
                 table.meta[key] = value
 
 
-def _convert_time_column(col, column_info):
+def _convert_time_column(col, column_info, global_info):
```

Comment:
I liked that before this code used only the `column_info`, since then one knew that that dict was the only thing one would ever need to look at to debug anything. Since you only use this to pass on the reference time, how about just passing that in, or setting `col_info['ref_time']` in the verification above?